FileOperations/FileOperations/FileDescription.cs

```diff
@@ -39,35 +39,32 @@ public void createFileAndAddText()
 			{
 				Console.WriteLine("Exception: " + e.Message);
 			}
-			finally
-			{
-				Console.WriteLine("Executing finally block.");
-			}
 		}
 
 		public void appendFile()
 		{
+			Console.WriteLine("File is being appended..");
 			// Adding this text in the file
 			try
 			{
-				using (StreamWriter sw = File.AppendText(Path))
+				using (StreamWriter sw = File.AppendText(path))
 				{
-					sw.WriteLine("This");
-					sw.WriteLine("is Extra");
-					sw.WriteLine("Text");
+					sw.WriteLine("\nThis is Extra Text");
 				}
 			}
 			catch (Exception e)
 			{
 				Console.WriteLine("Exception: " + e.Message);
 			}
-			
+
+			Console.WriteLine("File has been successfully appended..");
```

Comment:
This should be within try block. If an exception occurs, the program will still print "..successfully appended".